module-2/backend/week_one.md

```diff
@@ -4,31 +4,66 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 
 Note: When you're done, submit a PR. 
 
-1. List the five common HTTP verbs and what the purpose is of each verb.
+1. List the five common HTTP verbs and what the purpose is of each verb.  
+  *GET: Go to the path specified  
+  *POST:  Used when making a new object 
+  *PUT: Used for updating an object 
+  *DELETE: Used for deleting and object 
+  *PATCH: Not quite sure what this does   
+ 
+
 2. What is Sinatra?
+  Sinatra is a web frame work that allows you to make web applications. I think it is simpler than Rails.
 4. What is MVC?
+  Model, View, Controller
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
-6. What types of variables are accessible in our view templates without explicitly passing them?
+  They won't work with the MVC format if we don't.

```

Comment:
We could potentially make routes that don't follow convention that work with MVC, but working with conventions helps ensure that we can work easily with teammates who are new to a project.